dali/util/std_cufile.cc

```diff
@@ -115,6 +115,36 @@ void StdCUFileStream::HandleIOError(int64 ret) const {
   }
 }
 
+size_t StdCUFileStream::ReadGPUThread(uint8_t* gpu_buffer, size_t n_bytes, \
+                                      size_t buffer_offset, size_t file_offset) {
+  // effective pos:
+  size_t eff_pos = pos_ + file_offset;
+
+  // compute size
+  n_bytes = std::min(n_bytes, length_ - eff_pos);
+
+  // read data: backup n_bytes here and create a read-offset
+  ssize_t n_read = n_bytes;
+  off_t read_off = 0;
+  off_t buffer_off = buffer_offset;
```

Comment:
Do you need this alias here?

Reply:
you mean buffer_off?  I think that can go in favor of buffer_offset.

Reply:
It's incremented in line 138, so I believe it is fine to have it separated.

Reply:
But get the argument by value so it is fine to touch it.
Up to you.